api/routes/application/routes.py

```diff
@@ -63,7 +66,17 @@ def put(self):
 
     def submit(self, application_id):
         try:
-            application_id = submit_application(application_id)
-            return {"id": application_id}, 201
+            application = submit_application(application_id)
+            account = get_account(account_id=application.account_id)
+            application_with_form_json = get_application_with_forms(
+                application_id
+            )
+
+            Notification.send(
+                Config.NOTIFY_TEMPLATE_SUBMIT_APPLICATION,
+                account.email,
+                {"application": application_with_form_json},
+            )
```

Comment:
Looks good Adam!
 just a small comment
 Wondering if we could add the keys to send function , makes it easier to understand incase of debugging? 😛 
example: "to"=account.email

Reply:
That's ok. Not a big deal I was just being detail oriented for myself but I can see the function in external services/models/notification.py😆